app/error.tsx

```
@@ -1,15 +1,38 @@
 
+import { useState } from "react"
+
 import ProofCircle from "@/components/svgs/proof-circle.svg"
-import { HeroSection } from "@/components/ui/hero"
+import RefreshCW from "@/components/svgs/refresh-cw.svg"
+import { Button, ButtonLink } from "@/components/ui/button"
+import { Divider } from "@/components/ui/divider"
+
+import { cn } from "@/lib/utils"
 
 export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)